Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Controller.mo

```diff
@@ -0,0 +1,2275 @@
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uFanSpe(
+    final quantity="1",
+    final min=0,
+    final max=1) "Tower fan speed"
+    annotation (Placement(transformation(extent={{-840,-608},{-800,-568}}),
+      iconTransformation(extent={{-140,-230},{-100,-190}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TConWatSup(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature") if not closeCoupledPlant
+    "Condenser water supply temperature (condenser entering)"
+    annotation(Placement(transformation(extent={{-840,-680},{-800,-640}}),
+      iconTransformation(extent={{-140,-250},{-100,-210}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uIsoVal[nTowCel](
+    final min=fill(0, nTowCel),
+    final max=fill(1, nTowCel),
+    final unit=fill("1", nTowCel))
+    "Vector of tower cells isolation valve position"
+    annotation(Placement(transformation(extent={{-840,-728},{-800,-688}}),
+      iconTransformation(extent={{-140,-270},{-100,-230}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput watLev
+    "Measured water level"
+    annotation (Placement(transformation(extent={{-840,-760},{-800,-720}}),
+      iconTransformation(extent={{-140,-290},{-100,-250}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uTowSta[nTowCel]
+    "Vector of tower cell proven on status: true=running tower cell"
+    annotation(Placement(transformation(extent={{-840,-800},{-800,-760}}),
+      iconTransformation(extent={{-140,-310},{-100,-270}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput TChiWatSupSet(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Chilled water supply temperature"
+    annotation (Placement(transformation(extent={{800,630},{840,670}}),
+        iconTransformation(extent={{100,260},{140,300}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yChiWatPum[nChiWatPum]
+  if have_heaChiWatPum
+    "Chilled water pump status setpoint"
+    annotation(Placement(transformation(extent={{800,500},{840,540}}),
+      iconTransformation(extent={{100,230},{140,270}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yChiPumSpe[nChiWatPum](
+    final min=0,
+    final max=1,
+    final unit="1") "Chilled water pump speed setpoint"
+    annotation (Placement(transformation(extent={{800,460},{840,500}}),
+      iconTransformation(extent={{100,200},{140,240}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yChiDem[nChi](
+    final quantity=fill("ElectricCurrent",nChi),
+    final unit=fill("A", nChi)) if need_reduceChillerDemand
+    "Chiller demand setpoint to set through BACnet or similar "
+    annotation(Placement(transformation(extent={{800,400},{840,440}}),
+      iconTransformation(extent={{100,170},{140,210}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yChi[nChi]
+    "Chiller enabling status setpoint"
```

Comment:
"Enabling status setpoint" is not clear, "on/off command" or "enable signal" is clearer.

"Setpoint" should be reserved for the ***desired value of a controlled variable***.